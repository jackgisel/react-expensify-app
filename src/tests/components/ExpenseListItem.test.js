import React from 'react';
import { shallow } from 'enzyme';
import ExpenseListItem from '../../components/ExpenseListItem';
import expense from '../fixtures/expense';

test('should render ExpenseListItem with expense fixture', () => {
    const wrapper = shallow(<ExpenseListItem {...expense} />)
    expect(wrapper).toMatchSnapshot();
});