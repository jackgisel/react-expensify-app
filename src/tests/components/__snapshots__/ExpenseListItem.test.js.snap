// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render ExpenseListItem with expense fixture 1`] = `
<div>
  <Link
    replace={false}
    to="/edit/4"
  >
    <h3>
      in n out
    </h3>
  </Link>
  <p>
    $4.00
    -
    December 31st 1969
  </p>
</div>
`;
